Templates/set_theme_file.R

```diff
@@ -1,6 +1,6 @@
 # This file sets the theme for all the figures
 
-theme_set(theme_minimal(base_size = 12))
+figure_settings <- theme_set(theme_minimal(base_size = 12))
 
-my.discrete.colour()
-my.continous.colour()
\ No newline at end of file
+# my.discrete.colour()
```

Comment:
New untried solution

options(ggplot2.continuous.colour="viridis")
options(ggplot2.continuous.fill = "viridis")

Sets defaults so code will just use new colour scales


Discrete scales are a little more difficult but also possible - see 
https://stackoverflow.com/questions/53750310/how-to-change-default-color-scheme-in-ggplot2

Reply:
No this does not work... still makes red, green and blue dots